docs/draft/howto/enable-helm-chart-support.md

```diff
@@ -0,0 +1,415 @@
+# How to Enable Helm Chart Support Feature Gate
+
+## Description
+
+This document outlines the steps to enable the Helm Chart support feature gate in the OLMv1 and subsequently deploy a Helm Chart to a Kubernetes cluster. It involves patching the `operator-controller-controller-manager` deployment to enable the `HelmChartSupport` feature, setting up a network policy for the registry, deploying an OCI registry, and finally creating a ClusterExtension to deploy the metrics server helm chart.
+
+The feature allows developers and end-users to deploy Helm charts from OCI registries through the `ClusterExtension` API.
+
+## Demos
+
+[![Helm Chart Support Demo](https://asciinema.org/a/wEzsqXLDAflJvzSX7QP47GvLw.svg)](https://asciinema.org/a/wEzsqXLDAflJvzSX7QP47GvLw)
+
+
+## Enabling the Feature Gate
+
+To enable the Helm Chart support feature gate, you need to patch the `operator-controller-controller-manager` deployment in the `olmv1-system` namespace. This will add the `--feature-gates=HelmChartSupport=true` argument to the manager container.
+
+1.  **Create a patch file:**
+
+    ```bash
+    $ kubectl patch deployment -n olmv1-system operator-controller-controller-manager --type='json' -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--feature-gates=HelmChartSupport=true"}]'
+    ```
+
+2.  **Wait for the controller manager pods to be ready:**
+
+    ```bash
+    $ kubectl -n olmv1-system wait --for condition=ready pods -l control-plane=operator-controller-controller-manager
+    ```
+
+Once the above wait condition is met, the `HelmChartSupport` feature gate should be enabled in operator controller.
```

Comment:
There is a `run-experimental` make target that can be used to run this (of course it includes all other experimental features as well).
You might want to mention this alternative method.

Reply:
Good catch. Can we do a small revision after the PR is merged? I'm hesitant to add a change and kick out the one approval we have at this time.

Reply:
With the new `run-experimental`, I'm templated to say that our docs should _only_ talk about the experimental feature set, and not get into the details of enabling/disabling individual features.